site.conf

```diff
@@ -29,10 +29,6 @@
     geo_location = {
       show_altitude = false,
     },
-    remote_login = {
-      show_password_form = true,
-      min_password_length = 12,
-	  },
```

Comment:
While i think this is a sane choice, it does not impact the firmware size at all, as the code is still there, even with the password capability disabled.

Reply:
So will remove it for sanity reasons...